lib/pos_fee_estimator.go

```diff
@@ -10,6 +10,9 @@ import (
 )
 
 type PoSFeeEstimator struct {
+	// globalParams is the latest GlobalParams used by the PoSFeeEstimator. The fee estimation uses
+	// this whenever the cached blocks are refreshed
+	globalParams *GlobalParamsEntry
```

Comment:
Store the globalParams now, so we can move the feeEstimator's params over into GlobalParams in a later PR